lib/command-logger.coffee

```diff
@@ -162,6 +162,7 @@ class CommandLogger
   #
   # Returns the {String} format of the command time.
   formatTime: (time) ->
+    moment ?= require 'moment'
```

Comment:
@benogle this is called whenever a command is run right? So it would just slow down the first command run?